Bryce/actions catalog (#15528)
DataDog/documentation: content/en/workflows/actions_catalog/_index.md

kind: documentation
type: actioncatalog
disable_toc: true
cascade:
  disable_sidebar: true
  type: actioncatalog
---

> 

Reviewer: Please add `is_beta: true` to the frontmatter, as described in docs4docs: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Beta-callouts.

Author: Thanks, added

> <div class="alert alert-warning">Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>

Reviewer: Try using the {{ beta-callout }} shortcode.

Author: Are you sure? I had copied this from your doc to be consistent:
https://github.com/DataDog/documentation/blob/baf2c3e5072d9034d19678e8548fdd6f80554fb9/content/en/workflows/setup.md?plain=1#L11

I tried using the shortcode in a local build, and it looked pretty strange. Screenshot:
https://a.cl.ly/bLu5ZrDe

Removing all of the optional parameters listed in [docs4docs](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Beta-callouts) and adding `btn_hidden="true"` didn’t make any change to the appearance. This is the beta callout I ended up with, after initially starting with all of the parameters, and then removing them to try to alter the appearance:
```
{{< beta-callout url="#" btn_hidden="true" >}}
  Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.
{{< /beta-callout >}}
```
Maybe I’m using it incorrectly — if so, let me know if you can spot my mistake! Otherwise, I’m guessing it may be helpful to follow up on #websites for this?

Datadog Workflows provide actions that can be performed against your infrastructure and integrations. You can orchestrate and automate your end-to-end processes by linking together actions for cloud providers, Datadog integrations, Datadog itself, custom HTTP requests, and more. Read [build workflows][1] for more information on creating a workflow.

> Click on the buttons below to filter by different parts of Datadog Workflows. A description is available for each action, as well as the action's inputs, outputs, and parameters.

Reviewer: Removed the part about buttons, since that's a holdover from the security rules page. The action catalog doesn't have buttons.
```suggestion
See below a list of all available workflow actions. Click an action to see its description, inputs, outputs, and parameters.
```
